arbitrum-docs/launch-orbit-chain/orbit-sdk/how-tos/configuring-orbit-chain-with-sdk.md

```diff
@@ -0,0 +1,129 @@
+
+- `setInfraFeeAccount`: Sets the infrastructure fee account address, which receives all fees collected on the child chain. It is meant to receive the minimum base fee, with any amount above that going to the network fee account.
+
+- `setNetworkFeeAccount`: Sets the network fee account address. As mentioned, this address collects all fees above the base fee. Note that if you set this amount to the 0 address on your chain, all fees will be deposited into the infrastructure fee account.
+
+- `scheduleArbOSUpgrade`: If you plan to upgrade the <a data-quicklook-from="arbos">ArbOS</a> version of your chain, this method can help you schedule the upgrade. For a complete guide on this matter, please refer to the explanation of the [arbos upgrade](../../how-tos/arbos-upgrade.md).
+
+- `setChainConfig`: We discussed the chainConfig in the [Rollup deployment guide](deploying-rollup-chain-with-sdk.md#chain-config-parameter) in detail. If you wish to change any field of the `chainConfig`, you need to use this method on the child chain.
+
+### Getter Functions
+
+To read the child chain parameters, you can use these getter functions:
+
+- `getAllChainOwners`: Provides the list of all current chain owners.
+
+- `getNetworkFeeAccount`:  Returns the network fee account address.
+
+- `getInfraFeeAccount`:  Returns the infrastructure fee account address.
+
+- `isChainOwner`:  Allows you to check whether an address is on the list of chain owners.
+
+### Configuring the Child Chain Using the Orbit SDK
+
+In the Orbit SDK, we use the [Client Extension](https://viem.sh/docs/clients/custom#extending-with-actions-or-configuration) feature of Viem to extend the public client. In the Orbit SDK, we defined `arbOwnerPublicActions` to use it and extend the client on Viem. An example of creating a public client extended with arbOwner public actions is:
+
+
+```js
+import { createPublicClient, http } from 'viem';
+import { arbOwnerPublicActions } from '@arbitrum/orbit-sdk';
+
+const client = createPublicClient({
+  chain: arbitrumLocal,
+  transport: http(),
+}).extend(arbOwnerPublicActions);
+```
+
+With `arbOwnerPublicActions` and the public client in the Orbit SDK, we've added two new methods to the public clients:
+
+
+#### 1. arbOwnerReadContract 
+
+This method can be used to read the parameters of the child chain discussed in the [previous section](#getter-functions). An example of using this method with the `client` defined in the previous section is:
+
+```js
+  const result = await client.arbOwnerReadContract({
+    functionName: 'getAllChainOwners',
+  });
+```
+
+Changing the function names in the list in this [section](#getter-functions) will give you the other parameters.
```

Comment:
```suggestion
Changing the function names in the list in [the Getter functions section](#getter-functions) will give you the other parameters.
```
nit - specific link labels ([ref](https://docs.arbitrum.io/for-devs/contribute#style-conventions))